plugins/node/opentelemetry-instrumentation-hapi/README.md

```diff
@@ -25,38 +25,42 @@ To load a specific instrumentation (Hapi in this case), specify it in the regist
 
 ```js
 const { NodeTracerProvider } = require('@opentelemetry/node');
```

Comment:
replace `hapi-instrumentation` by `instrumentation-hapi` a few lines above (github doesn't allow to comment there)